### PR TITLE
First commit of Project Blueprint Docusaurus white label template

### PR DESCRIPTION
**Pull request by mcleo-d**
This is the first PR of the FINOS Project Blueprint Docusaurus white label template for FINOS projects to refer to when creating project related wikis and other local documentation.

### CODE REVIEW

**Comment by mcleo-d** [comment]
Hi @maoo - Can you review and accept this PR if appropriate as it will form the foundations of further FINOS work in the pipeline for @grizzwolf

File: website/build/Project Blueprint/blog/2016/03/11/blog-post.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...div><div class="navPusher"><div class="docMainWrapper wrapper"><div class="docsNavContainer" id="docsNav"><nav class="toc"><div class="toggleNav"><section class="navWrapper wrapper"><div class="navBreadcrumb wrapper"><div class="navToggle" id="navToggler"><div class="hamburger-menu"><div class="line1"></div><div class="line2"></div><div class="line3"></div></div></div><h2><i>›</i><span>Recent Posts</span></h2><div class="tocToggler" id="tocToggler"><i class="icon-toc"></i></div></div><div class="navGroups"><div class="navGroup"><h3 class="navGroupCategoryTitle">Recent Posts</h3><ul class=""><li class="navListItem"><a class="navItem" href="/blog/2017/10/24/new-version-1.0.0">New Version 1.0.0</a></li><li class="navListItem"><a class="navItem" href="/blog/2017/09/26/adding-rss">Adding RSS Support</a></li><li class="navListItem"><a class="navItem" href="/blog/2017/09/25/testing-rss">Adding RSS Support - RSS Truncation Test</a></li><li class="navListItem"><a class="navItem" href="/blog/2017/04/10/blog-post-two">New Blog Post</a></li><li class="navListItem navListItemActive"><a class="navItem" href="/blog/2016/03/11/blog-post">Blog Title</a></li></ul></div></div></section></div><script>
```
Comment (maoo) [inline website/build/Project Blueprint/blog/2016/03/11/blog-post.html]: `website/build` must be:
1. removed from Git
2. added to `.gitignore`

**Comment by maoo** [comment]
Wow, 18244 files changed :-)

Please make sure to never (ever) commit build files (ie `website/build`) and - most importantly - the `node_modules` folder.

Also, given that we don't have a preview feature to see the changes, it would be good to attach some screenshots of what been done.

Last, I don't understand why we would want to use Docusaurus to serve blog entries for our projects; this would lead to have one single blog "feed" for each project (which, on average, I'd expect to be almost empty); from a marketing PoV, I believe it would be much more useful to consolidate entries into www.finos.org/blog . So, my suggestion would be to disable blogs from the website blueprint.

**Comment by mcleo-d** [comment]
Yikes, sorry @maoo - Well done catching the `.gitignore` not being added to the project.

Let me go back and review why this file was not added as part of the original commit 👍

**Comment by mcleo-d** [comment]
Hi @maoo, 

I can confirm I have done the following according to your feedback ...

- Added the `.gitignore` which was excluded from the original commit. 
- Removed the `build` and `node_modules` folders and content from the repo.
- Provided the following screenshots of the build.

I can also confirm the blog is a feature of Docusaurus that's disabled in the `siteConfig.js` where the current config is outlined below ...

```
headerLinks: [
  {doc: 'finos-home', label: 'Docs'},   
  {doc: 'service-catalog', label: 'Use Cases'},
  {doc: 'finos-developer-identity', label: 'Get Involved'},
  {doc: 'service-catalog', label: 'Why Project Blueprint'},
  {blog: false}
]
```

**Comment by mcleo-d** [comment]
Screenshot 2020-02-04 at 13 59 36

**Comment by mcleo-d** [comment]
Screenshot 2020-02-04 at 13 59 48

**Comment by mcleo-d** [comment]
@maoo - I have now brought the repo in line with the `.gitignore` configuration so committed files have now been removed from repo 👍

**Comment by maoo** [comment]
Thanks for the update, looks much better now! I still see content in `website/blog` though, which I'd remove to keep things simple.

**Comment by mcleo-d** [comment]
After a conversation with @maoo, the Docusaurus blog functionality has been removed from FINOS Project Blueprint.

**Comment by maoo** [comment]
LGTM! TY @mcleo-d !!

**Comment by maoo** [comment]
@brooklynrob - the idea is to keep things simple and avoid adding items that are not part of the current website blueprint we used so far (fdc3, fo, alloy).

Also, and Ill let @toshaellison comment on it, I think it would be good to push ppl to contribute to finos.org/blog , instead of managing multiple semi-empty blog instances.

**Pull request merged**